### PR TITLE
OSX migrator fixes

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
Fixes #29 

@conda-forge/flann this is ready to be merged. 

The PR is based on #29, and in addition removing an obsolete if-statement for the build (https://github.com/conda-forge/flann-feedstock/pull/30/commits/d47fd7341804973e0bcbda8bd0f0ff599bf64039). Thanks @traversaro for the hint re this.

**Comment by Tobias-Fischer** [comment]
Ping @conda-forge/flann

File: recipe/meta.yaml [hunk 1]
```diff
@@ -48,8 +48,6 @@ test:
         {% endfor %}
         - test -d $PREFIX/include/flann                 # [not win]
         - if not exist %PREFIX%\\Library\\include\\flann exit 1      # [win]
-        - conda inspect linkages -p $PREFIX flann       # [not win]
```
Comment (wolfv) [inline recipe/meta.yaml:51]: was there a problem with running these?

**Comment by wolfv** [comment]
Looks good!

File: recipe/meta.yaml (hunk 1 above)
Comment (Tobias-Fischer) [reply recipe/meta.yaml:51]: These lines were deleted by the bot in https://github.com/conda-forge/flann-feedstock/pull/30/commits/36267f6d03e0eefd0d5009075d98873b12d4d0d6 - I guess they're deprecated but I'm not sure.

Comment (wolfv) [reply recipe/meta.yaml:51]: ok! perfect

**Pull request merged**